### ISSUE TITLE
Unable to delete all derived images from a single (or multiple) transformation(s)

### ISSUE THREAD

**Issue by ntucakovic** [cloudinary/cloudinary_php#72]
There's inconsistency with code documented at Admin API and Cloudinary PHP library.

Function `delete_resources` accepts `transformation` and not `transformations`.

If I pass `transformation`, it will be sent with request as is, and in the end will be ignored by Cloudinary API, causing all the images passed to have all of their derived images to be removed.

Workaround was to rename `transformation` to `transformations` at line 119 in Api.php 
```
return $this->call_api("delete", $uri, array_merge(array("public_ids"=>$public_ids), $this->only($options, array("keep_original", "invalidate", "transformations"))), $options);
```
But you must use string format for `transformations` parameter (meaning you have to only pass a single transformation), because array won't be transformed, since it's not `transformation` parameter it was expecting.

Anyhow I'd gladly make a patch but I can't find time to do so anytime soon. This is more for people who stumble on this issue.

Also, images are of type `fetch`, not `upload` in my case, if it means anything to maintainers.

I think this commit introduced the bug https://github.com/cloudinary/cloudinary_php/commit/0601bbe2295cac653e43a62bd7aea7f50c50f8d6

**Comment by roeeba** [comment]
Hi @ntucakovic, thank you very much for pointing this out. We'll make sure to fix this as soon as we can.

**Comment by const-cloudinary** [comment]
Fixed in #107
